README.md

```diff
@@ -1,8 +1,53 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nisrein Hinnawi . Jarvis Consulting
 
+I completed my Bachelor's degree in Computer Science from Jordan University of Science and Technology, and a Diploma degree in Computer Programming from Seneca College with honor, and I have a. During my studies, I have grown immense interest and enthusiasm in the field of research and development due to its innovative nature. Thinking out of the box, implementing new ideas, and finally developing those ideas into innovations are some of the main reasons which attract me into this field. I have the critical skills and knowledge which the market is looking for. I can work on different project platforms built with Java, C/C++, or C#. I have the ability to adapt and be an active team member in a short time, learn quickly new techniques and software as I have a strong and thorough understanding of programming languages, and meet stringent deadlines if required.
```

Comment:
I completed my Bachelor's degree in Computer Science from Jordan University of Science and Technology and a Diploma degree in Computer Programming from Seneca College with honor, and I have a. During my studies, I have grown immense interest and enthusiasm in the field of research and development due to its innovative nature. Thinking out of the box, implementing new ideas, and finally developing those ideas into innovations are some of the main reasons which attract me into this field. I can work on different project platforms built with Java, C/C++, or C#. I can adapt and be an active team member in a short time, learn quickly new techniques and software as I have a strong and thorough understanding of programming languages, and meet stringent deadlines if required.